TwitchDownloaderCLI/README.md

```diff
@@ -395,3 +413,9 @@ Default true boolean flags must be assigned: `--default-true-flag=false`. Defaul
 For Linux users, ensure both `fontconfig` and `libfontconfig1` are installed. `apt-get install fontconfig libfontconfig1` on Ubuntu.
 
 Some distros, like Linux Alpine, lack fonts for some languages (Arabic, Persian, Thai, etc.) If this is the case for you, install additional fonts families such as [Noto](https://fonts.google.com/noto/specimen/Noto+Sans) or check your distro's wiki page on fonts as it may have an install command for this specific scenario, such as the [Linux Alpine](https://wiki.alpinelinux.org/wiki/Fonts) font page.
+
+The list file for tsmerge can contain relative or absolute paths, but the path format differs from Windows to Linux/UNIX/MacOS (like volume paths or "/" instead of "\" to separate directories).
+The list must be a text file and describe one ts path per line.
+
+The concatenation made by tsmerge is not a raw (binary) concatenation, like `dd`, `cat` or `pv` would do on Linux, or `copy /b` on Windows.
+Instead, the streams have to be partially recoded, to keep its structure valid and playable. Using other software may not achieve this result.
```

Comment:
This function should be functionally equivalent to `cat`/`copy /b`, no? It quite literally opens a file, copies the bytes to a destination, then closes the file:
```cs
 await using (var fs = File.Open(partFile, FileMode.Open, FileAccess.Read, FileShare.Read))
{
    await fs.CopyToAsync(outputStream, cancellationToken).ConfigureAwait(false);
}
```

And I'm not sure why this says the ts files are partially recoded? We rely solely on FFmpeg for video encoding, and this function doesn't ever call FFmpeg.

Reply:
No it's not a raw concatenation, the resulting file has different size than the sum of its parts.

Reply:
Using `cat *.ts > cat.ts` under WSL I get the exact same result as I do from `CombineVideoParts`.
![image](https://github.com/lay295/TwitchDownloader/assets/72096833/1e844641-e914-44f3-b960-2f603585616d)
![image](https://github.com/lay295/TwitchDownloader/assets/72096833/87e6db33-7bd8-4093-ba25-942cdcc662fe)

In order to achieve this result with cat, however, I had to pad the filenames with zeroes so they were enumerated in the correct order.